src/H5Opline.c

```diff
@@ -110,6 +110,14 @@ H5FL_DEFINE(H5O_pline_t);
  *
  *-------------------------------------------------------------------------
  */
+
+#define VERIFY_LIMIT(p,s,l)                                                  \
+  if (p + s - 1 > l) {                                                       \
+    HGOTO_ERROR(H5E_RESOURCE, H5E_NOSPACE, NULL,                             \
```

Comment:
The library is inconsistant, but we generally make the major error category match the package, so this should be H5E_PLIST instead of H5E_RESOURCE.

Reply:
I've changed this and pushed the new version. 
Now it's inconsistent with the other error messages in the same function. These should probably be fixed separately.